x-pack/plugins/cases/public/containers/user_profiles/use_suggest_user_profiles.ts

```diff
@@ -31,16 +31,17 @@ export const useSuggestUserProfiles = ({
 
   const toasts = useToasts();
 
-  return useQuery<UserProfile[], ServerError>(
+  return useQuery<UserProfile[] | null, ServerError>(
     [
       USER_PROFILES_CACHE_KEY,
       USER_PROFILES_SUGGEST_CACHE_KEY,
       { name: debouncedName, owners, size },
     ],
     () => {
       if (isEmpty(name)) {
-        return [];
+        return null;
```

Comment:
If I try to return `undefined` I was getting an error that `onSuccess` can't receive undefined 🤷‍♂️